plugins/arDominionB5Plugin/modules/repository/templates/_uploadLimit.php

```diff
@@ -18,11 +18,15 @@
   </h5>
 
   <div class="card-body py-2">
+    <style <?php echo __(sfConfig::get('csp_nonce', '')); ?>>
+	#upload-limit-progress-div { height : 25px }
+	#upload-limit-progress-bar-div {width: <?php echo $sf_data->getRaw('diskUsageFloat'); ?>%}
```

Comment:
This one should stay as an inline style since the value is generated.

Reply:
BS class doesn't work for decimal percentages so will leave as is.